src/Microsoft.Health.Fhir.Tests.Common/TestFiles/R4/ProvenanceHeader.json

```diff
@@ -0,0 +1 @@
+﻿{"resourceType":"Provenance","id":"8a42c91e-dd16-4a9c-aed5-637076d8d7de","meta":{"versionId":"1","lastUpdated":"2021-01-28T21:22:47.984+00:00"},"text":{"status":"generated","div":"<div xmlns=\"http:\/\/www.w3.org\/1999\/xhtml\">procedure record authored on 27-June 2015 by Harold Hippocrates, MD Content extracted from XDS managed CDA Referral received 26-June as authorized by a referenced Consent.<\/div>"},"occurredPeriod":{"start":"2015-06-27","end":"2015-06-28"},"recorded":"2015-06-27T08:39:24+10:00","policy":["http:\/\/acme.com\/fhir\/Consent\/25"],"location":{"reference":"Location\/1"},"reason":[{"coding":[{"system":"http:\/\/snomed.info\/sct","code":"3457005","display":"Referral"}]}],"agent":[{"type":{"coding":[{"system":"http:\/\/terminology.hl7.org\/CodeSystem\/v3-ParticipationType","code":"AUT"}]},"who":{"reference":"Practitioner\/xcda-author"}},{"id":"a1","type":{"coding":[{"system":"http:\/\/terminology.hl7.org\/CodeSystem\/v3-ParticipationType","code":"DEV"}]},"who":{"reference":"Device\/software"}}],"entity":[{"role":"source","what":{"reference":"DocumentReference\/example","display":"CDA Document in XDS repository"}}]}
```

Comment:
Its hard to see from comparison, but if these are all the same you can just add a single file to `TestFiles/Normative/ProvenanceHeader.json` and it load for all versions